packages/tailwind/src/utils/css/sanitize-declarations.ts

```diff
@@ -0,0 +1,30 @@
+import type { AtRule, Root, Rule } from "postcss";
+
+const regex = /rgb\(\s*(\d+)\s*(\d+)\s*(\d+)(?:\s*\/\s*([\d%.]+))?\s*\)/g;
```

Comment:
can we find a better name that describes what regex is that? perhaps `declarationsRegex` or something